src/Command/DevServerCommand.php

```diff
@@ -19,6 +19,22 @@ public function __construct(WebpackCompiler $compiler)
         $this->compiler = $compiler;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('maba:webpack:dev-server')
+            ->setDescription('Run a webpack-dev-server as a separate process on localhost:8080')
```

Comment:
8080 is default port, it could be changed by configuration (`bin.dev_server.arguments` config value). But I guess if it's changed in config, developer should already know that this only describes the default one.